scripts/trainer.py

```diff
@@ -16,6 +18,44 @@
 basicConfig(level=INFO)
 logger = getLogger(__name__)
 
+TASKCLUSTER_DEFAULT_URL = "https://community-tc.services.mozilla.com"
+
+
+def get_taskcluster_options():
+    """Get the Taskcluster setup options according to current environment.
+
+    The current environment could be local.
+    """
+    options = taskcluster.optionsFromEnvironment()
+    proxy_url = os.environ.get("TASKCLUSTER_PROXY_URL")
+
+    if proxy_url is not None:
+        # Always use proxy url when available
+        options["rootUrl"] = proxy_url
+
+    if "rootUrl" not in options:
+        # Always have a value in root url
+        options["rootUrl"] = TASKCLUSTER_DEFAULT_URL
+
+    return options
+
+
+def create_task(task_definition):
+    options = get_taskcluster_options()
+    queue = taskcluster.Queue(options)
+
+    task_id = taskcluster.utils.slugId()
+
+    try:
+        queue.createTask(task_id, task_definition)
+        logger.info(
+            "Task created at https://community-tc.services.mozilla.com/tasks/%s",
+            task_id,
+        )
+    except taskcluster.exceptions.TaskclusterAuthFailure:
+        logger.exception("Failed to authenticate with Taskcluster")
```

Comment:
How do we handle the auth?

Reply:
> Have I done it correctly?

I do not know 😅 I did not use the API before. From the docs, it seems that the auth need to be done differentl: https://github.com/taskcluster/taskcluster/tree/main/clients/client-py#authentication-options

Reply:
You are right! I checked the `auth` credentials and they weren't present.  I will need to create them, I read in the documentation that-the authentication information passed with each API call is used to determine a set of scopes associated with the caller.
It's my first time using the API too. 😅 I am a little confused on the right credentials to create.